packages/common/src/store/notifications/selectors.ts

```diff
@@ -86,8 +86,9 @@ export const getNotificationUser = (
   notification: Notification
 ) => {
   if (
-    notification.type === NotificationType.Milestone &&
-    notification.achievement === Achievement.Followers
+    notification.type === NotificationType.TierChange ||
```

Comment:
nice add, so was this just missing in general?

Reply:
well we were filling the field on another part of the notification I believe